spring-cloud-contract-tools/spring-cloud-contract-gradle-plugin/src/main/groovy/org/springframework/cloud/contract/verifier/plugin/SpringCloudContractVerifierGradlePlugin.groovy

```diff
@@ -193,6 +194,7 @@ class SpringCloudContractVerifierGradlePlugin implements Plugin<Project> {
 	}
 
 	@CompileDynamic
+	@Deprecated
 	private void setPublications(def publishingExtension, TaskProvider<Task> stubsTask) {
 		publishingExtension.publications {
 			stubs(MavenPublication) {
```

Comment:
Maybe it's worth logging a warning message here that you're using a deprecated feature scheduled for removal and please create the publication task manually?